services/api/db/serializers.py

```diff
@@ -91,12 +94,14 @@ def validate(self, attrs):
         try:
             uid = urlsafe_base64_encode(force_bytes(user.id))
             token = PasswordResetTokenGenerator().make_token(user)
-        except Exception:
+            reset_link = reverse('reset-password-through-mail', kwargs={'uid': uid, 'token': token})
+        except ValidationError:
```

Comment:
```suggestion
        except ValidationError:
```
will the exception type ValidationError be able to handle all possible exceptions for the code block involving uid, token and reset_link

Reply:
Look at now I've added base Exception class to catch all possible issues.